examples/pxScene2d/src/node_modules/htmlparser/index.js

```diff
@@ -0,0 +1,7 @@
+var htmlparser = require("./lib/htmlparser");
```

Comment:
Please don't add this file - it duplicates node_modules/htmlparser/htmlparser.js . The fix had been done in #1559 , no need in this workaround anymore.